new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConfigDistributionStrategy.java

```diff
@@ -256,7 +256,7 @@ public void afterEach() {
 
     logger.info("Getting the list of servers using the listServers");
     String baseUri = "http://" + K8S_NODEPORT_HOST + ":" + serviceNodePort + "/clusterview/";
-    String serverListUri = "ClusterViewServlet?listServers=true";
+    String serverListUri = "ClusterViewServlet?user=" + ADMIN_USERNAME_DEFAULT + "&password=" + ADMIN_PASSWORD_DEFAULT;
     for (int i = 0; i < 5; i++) {
```

Comment:
call verifyMemberHealth method instead of iterating 5 times?

Reply:
The call is made to the Servlet just to see if the app is accessible after restart.
switched to dynamic wait instead of fixed iteration.